### PR TITLE
KAZOO-4064: Add api to add comments in any state

### PR DESCRIPTION
**Pull request by macpie**

### CODE REVIEW

File: applications/crossbar/doc/port_requests.md [hunk 1]
```diff
@@ -17,16 +17,16 @@ A port request can be in one of five states:
 ### List port requests
 
     curl -v -X GET \
-    -H "X-Auth-Token: {AUTH_TOKEN}" \
-    http://{SERVER}:8000/v2/accounts/{ACCOUNT_ID}/port_requests
+    -H "X-Auth-Token: {{AUTH_TOKEN}}" \
```
Comment (jamesaimonetti) [inline applications/crossbar/doc/port_requests.md]: Why the double brackets? Everywhere else uses single brackets.

File: applications/crossbar/doc/port_requests.md [hunk 2]
```diff
@@ -265,3 +265,112 @@ All requests are not paginated, with the exception of the `completed` state. Use
...
+    "status": "success"
+}
+```
+
+
+## Update port request comments
+
+### Request
+
+- Verb: `POST`
```
Comment (jamesaimonetti) [inline applications/crossbar/doc/port_requests.md]: This should really be a PUT to add a comment, and POST/DELETE to `/comments/{COMMENT_ID}` to edit/delete a comment.

You  may consider creating `cb_comments` to handle this (using the preceding `req_noun` to identify what thing the comments are associated with). Then we could add comments to users, devices, PBXes, etc.

File: applications/crossbar/doc/port_requests.md (hunk 1 above)
Comment (macpie) [inline applications/crossbar/doc/port_requests.md]: template engine uses double

File: applications/crossbar/doc/port_requests.md (hunk 2 above)
Comment (macpie) [inline applications/crossbar/doc/port_requests.md]: ok

**Pull request merged**